src/c_types.cr

```diff
@@ -0,0 +1,151 @@
+# This module exposes the various common platform-dependent arithmetic types
+# from the C language, to be used in lib bindings.
+#
+# For convenience, libs can import types from this module using aliases, so that
+# the types can be used inside the lib without fully qualified paths:
+#
+# ```
+# lib LibFoo
+#   alias Int = CTypes::Int
+#
+#   fun foo(x : Int) : Int     # okay
+#   fun bar(x : CTypes::Char*) # okay
+# end
+# ```
+module CTypes
```

Comment:
**thought:** I was wondering if `module C` could be an option as it allows to conveniently write `C::Char`. But that would likely cause confusion because `A`, `B`, `C` are commonly used as stub constants.

Importing with a local alias (`private alias C = ::CTypes`) would be of similar convenience.